training-net/src/Controllers/MoviesController.cs

```diff
@@ -20,8 +24,17 @@ public IActionResult Index()
             return View();
         }
 
-        public IActionResult Contact()
+        public IActionResult Create()
+        {
+            return View();
+        }
+
+        [HttpPost]
```

Comment:
for routing, we use the name of the route. here, for example: 
`[HttpPost("Create")]`